speculos/mcu/display.py

```diff
@@ -106,13 +107,13 @@ def __init__(self, model: str):
         self.current_screen_size = MODELS[model].screen_size
         self.screenshot = Screenshot(self.current_screen_size)
         # Init published content now, don't wait for the first request
-        if self.model == "stax":
+        if self.model == "stax" or self.model == "europa":
```

Comment:
Maybe could be replaced considering is_bagl (or use_bagl)?

Reply:
No, it only concerns Stax and Europa, not all NBGL targets

Reply:
Why? Nano devices don't use the refresh that's why?
Then we should probably use a variable that explain the specificity. Same in seproxyhal.py where some changes are linked I guess?

Reply:
The comments already explain everything